app/scripts/controllers/HomeCtrl.js

```diff
@@ -13,7 +22,10 @@
         vm.setActiveRoom = function (room) {
             document.getElementById("roomTitle").innerHTML = room.title;
```

Comment:
Why not just bind to the `activeRoom.title` in the template?

Reply:
You're right, I should have used ng-model there.  It would have saved me a lot of coding.